content/doc/developer/views/table-to-div-migration.adoc

```diff
@@ -3,10 +3,10 @@ title: Table to div layout migration
 layout: developerguide
 ---
 
-NOTE: This is a documentation for the incoming feature in the Jenkins core.
+NOTE: This is a documentation for a new feature in the Jenkins core.
 See jira:JENKINS-56109[Change Jenkins configuration UI from tables to divs] for details.
 
-Jenkins core will change its form layout from 'table' to 'div' soon.
+Jenkins core changed its form layout from 'table' to 'div' in 2.264 (this will be released in the next LTS).
```

Comment:
this is not the case - the next LTS is 2.263.1

Reply:
https://github.com/jenkins-infra/jenkins.io/pull/3989 for the correction.